packages/viewer/src/scenes/renderResult.ts

```diff
@@ -0,0 +1,40 @@
+import { Frame } from '../types';
+import { StreamApi } from '@vertexvis/stream-api';
+import { Result } from '../stream/result';
+import { StreamApiEventDispatcher } from '../stream/dispatcher';
+
+export interface RenderResultIds {
+  animationId?: string;
+  correlationId?: string;
+}
+
+export class RenderResult implements Result {
```

Comment:
nit, but how about we rename this to `CameraRenderResult` to prevent a future naming collision if we have another render method that has different events.